kalitectl.py

```diff
@@ -290,8 +293,11 @@ def manage(command, args=[], in_background=False):
         utility.prog_name = 'kalite manage'
         utility.execute()
     else:
-        thread = ManageThread(command, args=args)
-        thread.start()
+        if os.name != "nt":
+            thread = ManageThread(command, args=args, name=" ".join([command]+args))
+            thread.start()
+        else:
+            Popen([sys.executable, os.path.abspath(sys.argv[0]), "manage", command] + args, creationflags=CREATE_NEW_PROCESS_GROUP)
```

Comment:
Otherwise, we can leave this as is for 0.13. @MCGallaspy can you put a comment to try out having all platforms spawn another process, for 0.14?